src/scss/components/_index.scss

```diff
@@ -3,6 +3,7 @@
 @import "badge/index";
 @import "breadcrumb/index";
 @import "icon/index";
+@import "pagination/index";
```

Comment:
please alphabetize the imports

Reply:
Yes Ryan  done the changes and updated in git.